rules-reviewed/eap6/jboss-eap5/jboss-eap5-xml.windup.xml

```diff
@@ -139,12 +126,28 @@
         </rule>
 
 
+        <!-- The following 2 rules need to be deduplicated - see WINDUPRULE-164.
+             But currently we have no means for that. See WINDUP-1051 and WINDUP-1050. -->
+        <rule id="jboss-eap5-xml-02000">
+            <when>
+                <xmlfile public-id=".*JBoss.+DTD Java EE.+5.*"/>
+            </when>
+            <perform>
+                <classification title="JBoss 5.x EAR descriptor" effort="5" preventDuplication="title, any-tag">
+                    <description>
+                        A JBoss specific EAR descriptor allows extensions to Java EE EAR archives configuration.
+                    </description>
+                    <link title="JBoss DTD's" href="http://www.jboss.org/j2ee/dtd/"/>
+                    <tag>jboss-eap5</tag>
+                </classification>
+            </perform>
+        </rule>
         <rule id="jboss-eap5-xml-08000">
             <when>
                 <xmlfile matches="//*[local-name()='jboss-app']" in="jboss-app.xml"/>
```

Comment:
That's right, for these two rules it will prevent duplication. But not for the others, if there are any.
I mean - yes, let's do it that way here, but for general duplication prevention, skipping things touched by other rules by adding <not> could lead to quite complicated conditions. (Therefore my attempt in WINDUP-1050)